### ISSUE TITLE
rss broken

### ISSUE THREAD

**Issue by trobotham** [bitbot-irc/bitbot#184]
it appears that RSS has broken at some point:

```
~/bitbot$ python3 --version
Python 3.6.8
```

```
During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/bitbot/bitbot/src/utils/http.py", line 263, in _request
    "request_many failed for %s" % url)
src.utils.http.RequestManyException: request_many failed for https://www.reddit.com/r/irc/.rss
Task exception was never retrieved
future: <Task finished coro=<request_many.<locals>._request() done, defined at /home/bitbot/bitbot/src/utils/http.py:244> exception=RequestManyException('request_many failed for https://www.reddit.com/r/LinuxMalware/.rss',)>
Traceback (most recent call last):
  File "/home/bitbot/bitbot/src/utils/http.py", line 260, in _request
    response = await client.fetch(t_request)
TypeError: object TracebackFuture can't be used in 'await' expression

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/bitbot/bitbot/src/utils/http.py", line 263, in _request
    "request_many failed for %s" % url)
src.utils.http.RequestManyException: request_many failed for https://www.reddit.com/r/LinuxMalware/.rss
Task exception was never retrieved
future: <Task finished coro=<request_many.<locals>._request() done, defined at /home/bitbot/bitbot/src/utils/http.py:244> exception=RequestManyException('request_many failed for https://www.reddit.com/r/Malware/.rss',)>
Traceback (most recent call last):
  File "/home/bitbot/bitbot/src/utils/http.py", line 260, in _request
    response = await client.fetch(t_request)
TypeError: object TracebackFuture can't be used in 'await' expression

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/bitbot/bitbot/src/utils/http.py", line 263, in _request
    "request_many failed for %s" % url)
src.utils.http.RequestManyException: request_many failed for https://www.reddit.com/r/Malware/.rss
Task exception was never retrieved
future: <Task finished coro=<request_many.<locals>._request() done, defined at /home/bitbot/bitbot/src/utils/http.py:244> exception=RequestManyException('request_many failed for https://www.reddit.com/r/selfhosted/.rss',)>
Traceback (most recent call last):
  File "/home/bitbot/bitbot/src/utils/http.py", line 260, in _request
    response = await client.fetch(t_request)
TypeError: object TracebackFuture can't be used in 'await' expression
```

**Comment by jesopo** [comment]
Works on my instance, even with the same URL.

My assumption from the stacktrace is that the call to `.fetch()` fails and returns a weird wrapped Traceback?

just sanity check, do you know what version of Tornado you have installed?

**Comment by trobotham** [comment]
```
>>> print(tornado.version)
3.2.1
```

**Comment by jesopo** [comment]
fairly sure this is fixed by jesopo/bitbot@7d958be513ae44870b28b88e20c86becc5ca2d32
